lib/tasks/temporary/hacktoberfest_badges.rake

```diff
@@ -0,0 +1,34 @@
+# TODO: Will be removed my @mstruve once badges are rewarded for 2020
+# rubocop:disable  Metrics/BlockLength
+task award_hacktoberfest_badges: :environment do
+  badge_id = Badge.find_by(slug: "hacktoberfest-2020")&.id
+  user_ids = UserSubscription.where(
+    user_subscription_sourceable_id: 529_344,
+    created_at: 2.hours.ago..Time.current,
+  ).pluck(:subscriber_id)
+
+  User.where(id: user_ids).includes(:identities).find_each do |user|
+    next if user.badges.exists?(id: badge_id)
+
+    identity = user.identities.github.first
+    next unless identity
+
+    github_user_id = identity.uid
+    response = HTTParty.get(
+      "https://hacktoberfest.digitalocean.com/api/state/#{github_user_id}",
+      headers: { "Authorization" => ENV["HACKTOBERFEST_AUTH_KEY"] },
+    )
+    if response.parsed_response == "completed"
```

Comment:
There are two response options that I have seen in the manual run I am doing, "registered" which we do not award for, and "completed" which we do award the badge for. 

I am running the first 6k manually so I would expect any possible rate-limiting will show up if there is an issue on that front.